Sync/Synchronizers/ClientsSynchronizer.swift

```diff
@@ -22,6 +24,17 @@ public protocol Command {
 // We need a way to log out the account.
 // So when we sync commands, we're gonna need a delegate of some kind.
 public class WipeCommand: Command {
+    var guid: GUID?
+    var clientGuid: GUID?
+    var modified: Timestamp?
```

Comment:
So… with modified and guid removed, this gets simpler. And I would argue that a command does not own its client GUID — you can send the same command to multiple clients, and the only place we pair up a command and a destination client GUID is in storage. So I'd revert the changes here.